onnxruntime/core/common/cpuid_info.h

```diff
@@ -122,8 +122,16 @@ class CPUIDInfo {
 
   void X86Init();
 #elif defined(CPUIDINFO_ARCH_ARM)
-  // Now the following var is only used in ARM build, but later one we may expand the usage.
+// Now the following var is only used in ARM build, but later one we may expand the usage.
```

Comment:
thanks @yf711 for reporting this.

question - Jetson uses gcc by default, but do we need to accommodate for other compilers as well?